.buildkite/scripts/check-storybook-accessibility.ts

```diff
@@ -60,20 +62,35 @@ const main = async () => {
     )}&selectedStory=${encodeURIComponent(name)}`
   })
 
-  const violations = await getViolations(storybookExampleUrls, page)
+  const examples: ExampleWithViolations[] = await examplesWithViolations(
+    storybookExampleUrls,
+    page
+  )
+
+  const violationCount = examples.reduce(
+    (tally, example) => tally + example.violations.length,
+    0
+  )
 
   await page.close()
   await browser.close()
 
-  if (violations.length > 0) {
+  if (violationCount == 0) {
+    console.log("No accessibility violations found")
+  } else if (violationCount > passableViolationCount) {
     console.log("Accessibility violations found:")
-    violations.forEach(analysis => {
-      console.log(analysis)
+    examples.forEach(example => {
+      console.log(example)
     })
+    console.log(
+      `More accessibility violations were found than the current allowable limit of ${passableViolationCount}. This likely means that a new accessibility violation has been added to the code base.`
+    )
     process.exit(1)
   } else {
```

Comment:
Fail here too (give a nice 🌈  error message, asking engineer to reduce passableViolationCount)